packages/components/src/AtlantisPortalContent/types.ts

```diff
@@ -0,0 +1,4 @@
+export interface AtlantisPortalContentProps {
+  /** The content that will be rendered inside a portal. */
```

Comment:
Minor comment on the types files.

It seems like we have been doing `<componentName>.types.ts` for our types files in components. I think this is preferable because it makes searching & finding the types easier. It also keeps it consistent with other types files

Reply:
My bad, I must have missed that when reviewing it in the Card PR